rmw_fastrtps_shared_cpp/src/rmw_service.cpp

```diff
@@ -138,4 +138,18 @@ __rmw_destroy_service(
   RCUTILS_CAN_RETURN_WITH_ERROR_OF(RMW_RET_ERROR);  // on completion
   return final_ret;
 }
+
+rmw_ret_t
+__rmw_service_get_actual_qos(
+  const rmw_service_t * service,
+  rmw_qos_profile_t * qos)
+{
+  auto info = static_cast<CustomServiceInfo *>(service->data);
+
+  eprosima::fastdds::dds::DataReader * fastdds_dr = info->request_reader_;
```

Comment:
same here.

Reply:
Done in 7004b814c49b4b118fd2894e54ca69b9e37d27bb